npm-shrinkwrap.json

```diff
@@ -2063,6 +2068,460 @@
     "fs.realpath": {
       "version": "1.0.0"
     },
+    "fsevents": {
+      "version": "1.1.1",
+      "optional": true,
```

Comment:
The new shrinkwrap script now brings this in, but this version of Node should be able to handle skipping this, on platforms that don't support this.